### PR TITLE
ADAPT-4684: Scroll wrapper for table

### PR DESCRIPTION
**Pull request by Lyafel**
# Summary
- Adding the scroll for the tables that don't fit in the page width.
- This comes with the need for a wrapper to the custom content tables, if solution works, it would have to go to wherever the docs for the tables went.
 
# Review By (Date)
- When does this need to be reviewed by?

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to test-items/elena/table-test
3. Verify the scrollable table

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
@Lyafel Do you have a link to a test page somewhere? 😃

**Comment by Lyafel** [comment]
> @Lyafel Do you have a link to a test page somewhere? 😃

Yes! But I've once again lost the battle with netlify! The page doesn't appear on the build, I've published it, cleared cache and rebuilt the preview and nothing :'(
https://app.storyblok.com/#!/me/spaces/78141/stories/0/0/105493408 this is the test page

**Comment by sherakama** [comment]
> > @Lyafel Do you have a link to a test page somewhere? 😃
> 
> Yes! But I've once again lost the battle with netlify! The page doesn't appear on the build, I've published it, cleared cache and rebuilt the preview and nothing :'( [app.storyblok.com/#!/me/spaces/78141/stories/0/0/105493408](https://app.storyblok.com/#!/me/spaces/78141/stories/0/0/105493408) this is the test page

That space is the PRODUCTION space and your pull request is into the development branch wich would be on the DEV space here: https://app.storyblok.com/#!/me/spaces/122401/dashboard

**Comment by Lyafel** [comment]
> > > @Lyafel Do you have a link to a test page somewhere? 😃
> > 
> > 
> > Yes! But I've once again lost the battle with netlify! The page doesn't appear on the build, I've published it, cleared cache and rebuilt the preview and nothing :'( [app.storyblok.com/#!/me/spaces/78141/stories/0/0/105493408](https://app.storyblok.com/#!/me/spaces/78141/stories/0/0/105493408) this is the test page
> 
> That space is the PRODUCTION space and your pull request is into the development branch wich would be on the DEV space here: https://app.storyblok.com/#!/me/spaces/122401/dashboard

That explains it, didn't know there's a different space for giving! I can't access the link, I only have the prod space in my list, could you please add me?

**Comment by yvonnetangsu** [comment]
@Lyafel you've been added to the DEV - OOD Giving space 😃

**Comment by Lyafel** [comment]
> @Lyafel you've been added to the DEV - OOD Giving space 😃

Thank you :) here's the test page https://deploy-preview-248--adapt-giving.netlify.app/test-items/elena/table-test/

File: src/scss/elements/_table.scss [hunk 1]
```diff
@@ -77,3 +77,16 @@ table:not(.life-income-gifts) {
     background:  $su-color-fog-light;
   }
 }
+
+// Wrapper that adds the horizontal scroll it the table is wider than the parent
+.table-scroll {
+  overflow: auto;
+  max-width: 100%;
+  margin-bottom: 7.2rem;
```
Comment (yvonnetangsu) [inline src/scss/elements/_table.scss:85]: This looks good to me, @Lyafel ! Only questions is - any reason why adding this big margin-bottom only on mobile?

Comment (Lyafel) [reply src/scss/elements/_table.scss:85]: @yvonnetangsu I'm not. The table has spacing on deesktop, I added the spacing to the wrapper and removed it from the tablet if wrapped in table-scroll.

**Review by yvonnetangsu — APPROVED** [review]
Thanks, @Lyafel ! This looks great. GTG 😄

**Pull request merged**